perfkitbenchmarker/providers/openstack/os_virtual_machine.py

```diff
@@ -190,6 +190,17 @@ def WaitForBootCompletion(self):
         if self.hostname is None:
             self.hostname = resp[:-1]
 
+    @vm_util.Retry(poll_interval=5, max_retries=-1, timeout=300,
+                   log_errors=False,
+                   retryable_exceptions=(
+                       errors.Resource.RetryableDeletionError,))
```

Comment:
A bit of a tangent, but it looks like we have a lot of places where retryable_exceptions lists only one type, and it looks icky as a tuple. As far as I can tell, Retry should be able to work if you just pass `retryable_exceptions=errors.Resource.RetryableDeletionError`, but the Retry docstring doesn't explicitly say so. Maybe we should allow this.